### PR TITLE
Fixed 0026036: Wrong permission query

### PR DESCRIPTION
**Pull request by smeyer-ilias**
The export tab was only visible for users assigned to the administrator role, since rbacsystem was used instead of ilAccess and the "ref_id-parameter" was emtpy. 

I moved all rbacsystem calls to ilAccess in function getTabs()

### CODE REVIEW

File: Modules/ScormAicc/classes/class.ilObjSAHSLearningModuleGUI.php [hunk 1]
```diff
@@ -768,15 +770,15 @@ function getTabs()
 		}
 
 		// export
-		if ($rbacsystem->checkAccess("edit_permission", "", $this->object->getRefId()))
+		if ($ilAccess->checkAccess("edit_permission", "", $this->object->getRefId()))
```
Comment (smeyer-ilias) [inline Modules/ScormAicc/classes/class.ilObjSAHSLearningModuleGUI.php:773]: Is the permission "edit_permission" correct here? 
Normally "write" permission is sufficient.

**Comment by kergomard** [comment]
@Uwe-Kohnle : Could you please have a look at this PR and either check in with @smeyer-ilias to resolve the conflicts and merge or state your concerns and close it. Thank you very much!

**Comment by Amstutz** [comment]
@Uwe-Kohnle : Hi Uwe, the TB regularly monitors the list of PRs for old ones. This one here popped up again. Any Feedback? Note, you might also close this or ask for funding if an analysis seems very time consuming.

**Comment by Uwe-Kohnle** [comment]
Hi Stefan,
i also replaced edit_permission to write for offline_mode_manager
see:
https://github.com/ILIAS-eLearning/ILIAS/commit/25f2e68d9e950c639bec5d22969942dadc36a261

**Pull request closed without merging**